File: Code/Websites/DanpheEMR/Services/Appointment/DTO/PatientLastVisitContext_DTO.cs
using System;

namespace DanpheEMR.Services.Appointment.DTO
{
    public class PatientLastVisitContext_DTO
    {
       public int PatientId { get; set; }
       public int PatientVisitId { get; set; }
       public string VisitCode { get; set; }
       public int SchemeId { get; set; }
       public int PriceCategoryId { get; set; }
        public DateTime VisitDate { get; set; }
       public string VisitType { get; set; }
       public int DepartmentId { get; set; }
       public int? PerformerId { get; set; }
       public bool IsCurrentlyAdmitted { get; set; }
       public DateTime? DischargeDate { get; set; }
    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/appointments/shared/dto/patient-lastvisit-context.dto.ts
//sud:8Sept'21--common class to get patient's latest visit context.

export class PatientLatestVisitContext_DTO {
    public PatientId: number = null;
    public PatientVisitId: number = null;
    public VisitCode: string = null;
    public SchemeId: number = null;
    public PriceCategoryId: number = null;
    public VisitDate: string = null;
    public VisitType: string = null;
    public DepartmentId: number = null;
    public PerformerId: number = null;
    public IsCurrentlyAdmitted: boolean = false;
    public DischargeDate: string = null;
}